src/STEMMUS_SCOPE.m

```diff
@@ -443,7 +443,7 @@
 
                 switch options.calc_ebal
                     case 1
-                        [iter, fluxes, rad, thermal, profiles, soil, RWU, frac] ...
+                        [iter, fluxes, rad, thermal, profiles, soil, RWU, frac, sfactor, psiLeaf] ...
```

Comment:
Please follow [Matlab guidelines](http://cnl.sogang.ac.kr/cnlab/lectures/programming/matlab/Richard_Johnson-MatlabStyle2_book.pdf) on variable names, see page 7 to choose variable names here. "The names of variables should document their meaning or use." For example, `soilWaterStressFactor` instead of `sfactor`.

Reply:
Thanks for your suggestions. I revise the names of variables accordingly.